### DISCUSSION TITLE
compliance trivy 0.35 vs 0.42

### DISCUSSION THREAD

**Discussion by senemamourr** [aquasecurity/trivy#4563] [Q&A]
### Question

Hello,
with version 0.35 of trivy we can launch the appliance without creating a job at the kubernetes cluster level but with the new versions (> 0.35), it creates a job that creates a pod in a given namespace to perform the scans.
Please can you tell me what  are the added value the new versions bring by creating a job on the cluster ?

### Target

Kubernetes

### Scanner

Vulnerability

### Output Format

None

### Mode

None

**Comment by knqyf263** [comment]
@chen-keinan Can you please take a look?

**Reply by chen-keinan** [reply]
@senemamourr this new capabilities is part of infra assessment and compliance reports functionality.
The scan-job purpose is to  scan the k8s Nodes for config various  issues which require access to host file system.
such as api-server and kubelet configs and many more.

these infra checks enable trivy to assess how harden is your cluster with in term of security (nsa,cis and more)

**Reply by senemamourr** [reply]
thank you @chen-keinan for your answer.
so with 0.35 i do not have infrastructure assessment, only manifest scan and with the new version the job need to access to hospath for scanning infras config ?
we actually  use psa that prohibit access to node's file system from a pod. 
is there a way to configure the job to comply with our security policy or do we have to disable the psa for infrastructure scanning?
best regards,

**Reply by chen-keinan** [reply]
@senemamourr yes with new version the job mount to host filesystem 
currently the scan-job can't be configure via `trivy` CLI  take a look at the [job tamplate](https://github.com/aquasecurity/trivy-kubernetes/blob/main/pkg/jobs/template/node-collector.yaml) ) and let me know if you have any thoughts
